deps/asio/build-asio

```diff
@@ -28,6 +28,6 @@ CSUM=${ASIO_CSUM}
 download
 
 cd $DEP_DIR
-rm -rf asio*
+#rm -rf asio*
```

Comment:
It is deleting the `asio*` folders from the `$DEP_DIR` directory (note the cd command right above). It does so because the line right after will uncompress the tar archive from the `$DL` folder